exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -145,93 +65,26 @@ solution yet! The use of an
 in this situation is designed to remind us of exactly this fact. It effectively
 says: "Your Code Goes Here!".
 
-Delete the contents of line 4 and replace it with the following:
+Delete the contents of line 4 and replace it with:
 
 ```java
-return null;
-```
-
-Now run `gradle test` again.
-
-You should see a new error this time. Don't worry though, that still
-represents forward progress!
-
-```
-GreeterTest > testThatGreeterReturnsTheCorrectGreeting FAILED
-    java.lang.AssertionError: expected:<Hello, World!> but was:<null>
-```
-
-...is read: "Within the test class named `GreeterTest`, 
-the test method `testThatGreeterReturnsTheCorrectGreeting` did not pass because
-the solution did not satisfy an assertion." 
-
-Apparently, our test was expecting to see the string "Hello, World!", but it
-received the value `null` instead.
-
-The last line of the stack trace tells us exactly where this "unsatisfied
-assertion" lives:
-
-```
-        at testThatGreeterReturnsTheCorrectGreeting(GreeterTest.java:9)
-```
-Looks like the mismatch was discovered on line 9 in the test file.
-
-Knowing these two facts:
-
-1. that the return value was not what was expected, and
-2. that the failure was on line 9 of the test file,
-
-we can turn this failure into success.
-
-## Step 3: Fix the test!
-
-Open `src/test/java/GreeterTest.java` and go to line 9. It reads:
-
-```java
-assertEquals("Hello, World!", new Greeter().getGreeting());
-```
-
-The test is expecting that the method `getGreeting()` returns "Hello, World!".
-Instead, `getGreeting()` is returning `null`. Let's fix that.
-
-Open `src/main/java/Greeter.java`. It should look like this:
-
-```java
-class Greeter {
-    String getGreeting() {
-        return null;
+    public String getGreeting() {
+        return "Hello, World!";
     }
-}
 ```
 
-Remove the `return null` that we previously added, and instead return the
-string our test was expecting:
+## Step 2: Run the Tests
 
-```java
-    String getGreeting() {
-        return "Hello, World!";
-    }
-```
+Once again, tests can be 
 
-Save the file and run the tests again:
+After making corrections and implementing your solution, ensure to run the program by selecting the "Run Tests"
 
-```
-$ gradle test
-:compileJava
-:processResources UP-TO-DATE
-:classes
-:compileTestJava
-:processTestResources UP-TO-DATE
-:testClasses
-:test
-GreeterTest > testThatGreeterReturnsTheCorrectGreeting PASSED
-BUILD SUCCESSFUL
-Total time: 11.717 secs
-```
+button on the online editor. If the tests fails, that's ok! Simply refactor your code and test again.
```

Comment:
Lines 80-82 should have no blank lines in the middle. You can put this into the same line, or alternatively put each sentence in a new line, but without blank lines in the middle.

Reply:
Removed the new lines in between 80-82. It's one small paragraph.